modules/oc-mirror-about-ts-v2.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-about-ts-v2_{context}"]
+== Troubleshooting oc-mirror plugin v2 issues
+
+In oc-mirror plugin v2, there are specific behaviors to handle errors during the mirroring process, especially when issues arise with operator images, additional images, or release images. Understanding how the tool behaves in these scenarios can help users troubleshoot and resolve issues more effectively.
+
+*Key Behaviors of oc-mirror plugin v2*:
+
+* Mirroring Continuation:
+
+When errors occur while mirroring operator or additional images, oc-mirror plugin v2 continues to mirror other images instead of halting the entire process. However, if there are errors during the mirroring of release images, the process is aborted.
```

Comment:
May be worth adding `continues to mirror other images instead of halting the entire process and generates a txt file with all the images and why it failed to mirror so that user could review and take a proper action`

s/errors/error